.golangci.yml

```diff
@@ -200,7 +200,7 @@ linters-settings:
       # - filepathJoin
       # - whyNoLint
   gocyclo:
-    min-complexity: 165
+    min-complexity: 170
```

Comment:
The linter gives you a hint (to maybe move your code into a separate function). Don't ignore it.

Reply:
It seems the refactoring didn't help in this case. We really need to push for the resolution of [cri-o/cri-o issue #7275](https://github.com/cri-o/cri-o/issues/7275). @hasan4791, do you need any help with moving the container creation pieces to `internal/factory`?